app/cars/page.jsx

```diff
@@ -0,0 +1,92 @@
+'use client';
+
+import { useSearchParams } from 'next/navigation';
+import React, { useState, useEffect } from 'react';
+import Link from 'next/link';
+import { RoundedButton } from '@/components/buttons';
+import { colors } from '@/utilities/common';
+import PropTypes from 'prop-types';
+import arrow from '@/assets/images/arrow.png';
+import { Api } from '@/utilities/common';
+import axios from 'axios';
+
+import style from './page.module.css';
+
+const ItemContainer = ({ id, image, name, cost_per_day }) => {
+  const [value, setValue] = useState(5);
+
+  const handleInputChange = (e) => {
+    const inputValue = e.target.value;
+    const intValue = parseInt(inputValue.replace(/\D/g, ''), 10);
+    setValue(intValue);
+  };
+
+  return (
+    <div className={style.details_itemContainer}>
+      <img src={image} alt='car' className={style.details_image} />
+      <div className={style.details_table_container}>
+        <h3>{name}</h3>
+        <table className={style.car_table}>
+          <tr>
+            <td>Cost</td>
+            <td className={style.table_right_item}>${cost_per_day}</td>
+          </tr>
+          <tr>
+            <td>Days</td>
+            <td className={style.table_right_item}>
+              <input
+                type='text'
+                value={value}
+                onChange={handleInputChange}
+                placeholder='Enter an integer'
+              />
+            </td>
+          </tr>
+          <tr>
+            <td>Total</td>
+            <td className={style.table_right_item}>${cost_per_day * value}</td>
+          </tr>
+        </table>
+        <h2 className={style.details_separator}>..................</h2>
+        <Link href={`/reservation?id=${id}`}>
+          <RoundedButton color={colors.green}>RESERVE</RoundedButton>
+        </Link>
+      </div>
+      <div className={style.details_button_container}>
+        <Link href={`/home`} className={style.details_button_link}>
+          <img src={arrow.src} alt='left icon' height='5px' />
+        </Link>
+      </div>
+    </div>
+  );
+};
+
+export default function page() {
```

Comment:
- [ ]  Please, kindly refactor this function to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const page = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪